README.md

```diff
@@ -95,6 +95,13 @@ Set your inital password:
     >>> set_password <PASSWORD>
     >>> unlock <PASSWORD>
 
+**IMPORTANT:** The cli_wallet or API interfaces to the witness node wouldn't be fully functional unless the witness node is fully synchronized with the blockchain, the cli_wallet command "info" will show how far are you from the live current block of the blockchain, you can compare the result "head_block_num" with the current block number appearing [here](https://wallet.bitshares.org/#/explorer/blocks).
```

Comment:
Please replace the ',' with '.' (short sentences are easier to read).
"how far are you" -> "how far you are"
Please use backticks to quote commands and other technical keywords (i. e. `info`, `head_block_num`).
Instead of comparing `head_block_num` with an explorer, it is easier to check `head_block_age` (provided your local machine's clock is correct).